app.js

```diff
@@ -0,0 +1,116 @@
+
+// On load, hide battlefield
+$("#battlefield").hide();
+
```

Comment:
To avoid possible flash of content before the js loads, you could hide this with a class in CSS then remove the class when you want to show the battlefield